pages/pools/createpool.tsx

```diff
@@ -0,0 +1,40 @@
+import type { NextPage } from "next"
+
+const CreatePool: NextPage = () => {
+  const style = {
+    wrapper: "w-screen flex flex-col items-center",
+    card: "flex flex-col bg-gray-900 w-[30rem] rounded-2xl p-4 mt-4",
+    cardHead: "font-semibold text-xl text-gray-200 px-2",
+    createPoolHeading: "",
```

Comment:
Consider adding some border radius to the table and buttons. It looks too sharp

Reply:
Done